acquia_lift.install

```diff
@@ -203,6 +203,32 @@ function _acquia_lift_build_menu($menu_name) {
   );
   menu_link_save($item);
 
+  // Create a link to the all campaigns page.
+  $item = array(
+    'link_title' => 'All campaigns',
+    'link_path' => 'admin/structure/personalize',
+    'menu_name' => $menu_name,
+    'options' => array(
+      'attributes' => array(
+        'aria-role' => 'button',
+        'aria-pressed' => 'false',
+        'class' => array(
+          'acquia-lift-campaign-all',
```

Comment:
You are right, I think they are just adding useless overhead. My previous colleagues picked consistency over cleanliness, and I really don't have a strong feeling towards either (other than clarifying the reasoning). In this case, let's use the Lift team's current standard and remove the "-all" classes.